Cartfile.resolved

```diff
@@ -1 +1 @@
-github "marekcirkos/RoutingHTTPServer" "v1.0.1"
+github "shvul/RoutingHTTPServer" "v1.0.2"
```

Comment:
What exactly has been changed in this module that is has to be updated?

Reply:
Previous version doesn't have tvOS target, so Carthage cannot build it for our purposes. Also, [CocoaHTTPServer](https://github.com/shvul/CocoaHTTPServer) lib was updated. I added small fix for tvOS compiling.

Reply:
The original repo has not been updated for a long time. @marekcirkos made a fork for his fixes. Do you suggest to make pr to his fork?

Reply:
yes, this makes sense, because he is anyway the only person who could approve and merge this PR

Reply:
@mykola-mokhnach here it is: https://github.com/marekcirkos/RoutingHTTPServer/pull/1
Let's continue when @marekcirkos merge it and create new release